products/appengine/api.yaml

```diff
@@ -743,8 +745,112 @@ objects:
+        name: 'handlers'
+        description: |
+          An ordered list of URL-matching patterns that should be applied to incoming requests.
+          The first matching URL handles the request and other request handlers are not attempted.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+          - !ruby/object:Api::Type::String
+            name: 'urlRegex'
+            description: |
+              URL prefix. Uses regular expression syntax, which means regexp special characters must be escaped, but should not contain groupings.
+              All URLs that begin with this prefix are handled by this handler, using the portion of the URL after the prefix as part of the file path.
+          - !ruby/object:Api::Type::Enum
+            name: 'securityLevel'
+            required: false
+            description: |
+              Security (HTTPS) enforcement for this URL.
+            values:
+              - :SECURE_DEFAULT
+              - :SECURE_NEVER
+              - :SECURE_OPTIONAL
+              - :SECURE_ALWAYS
+          - !ruby/object:Api::Type::Enum
+            name: 'login'
+            description: |
+              Methods to restrict access to a URL based on login status.
+            required: false
+            values:
+              - :LOGIN_OPTIONAL
+              - :LOGIN_ADMIN
+              - :LOGIN_REQUIRED
+          - !ruby/object:Api::Type::Enum
+            name: 'authFailAction'
+            description: |
+              Actions to take when the user is not logged in.
+            required: false
+            values:
+              - :AUTH_FAIL_ACTION_REDIRECT
+              - :AUTH_FAIL_ACTION_UNAUTHORIZED
+          - !ruby/object:Api::Type::Enum
+            name: 'redirectHttpResponseCode'
+            description: |
+              30x code to use when performing redirects for the secure field.
+            required: false
+            values:
+              - :REDIRECT_HTTP_RESPONSE_CODE_301
+              - :REDIRECT_HTTP_RESPONSE_CODE_302
+              - :REDIRECT_HTTP_RESPONSE_CODE_303
+              - :REDIRECT_HTTP_RESPONSE_CODE_307
+          - !ruby/object:Api::Type::NestedObject
+            name: 'script'
+            # TODO (mbang): Exactly one of script, staticFiles, or apiEndpoint must be set
+            description: |
+              Executes a script to handle the requests that match this URL pattern.
+              Only the auto value is supported for Node.js in the App Engine standard environment, for example "script:" "auto".
+            properties:
+            - !ruby/object:Api::Type::String
+              name: 'scriptPath'
+              required: true
+              description: |
+                Path to the script from the application root directory.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'staticFiles'
+            # TODO (mbang): Exactly one of script, staticFiles, or apiEndpoint must be set
```

Comment:
Want to add this now?

Reply:
unfortunately, we still can't, because it's a child of an array